backend/nodes/image_util_nodes.py

```diff
@@ -26,9 +26,9 @@ def __init__(self):
         self.inputs = [
             ImageInput("Base"),
             ImageInput("Overlay A"),
-            SliderInput("Opacity A", default=50, min_val=0, max_val=100),
+            SliderInput("Opacity A", minimum=0, maximum=100, default=50),
             ImageInput("Overlay B ", optional=True),
-            SliderInput("Opacity B", default=50, min_val=0, max_val=100, optional=True),
+            SliderInput("Opacity B", minimum=0, maximum=100, default=50, optional=True),
```

Comment:
@joeyballentine Does it even make sense for a slider to be optional? They have a default value (always, I think), so they will always have a value assigned to them.

Reply:
I'll make this change in the PR with the other changes to this node.